tests/foreman/cli/test_syncplan.py

```diff
@@ -273,3 +319,271 @@ def test_verify_bugzilla_1261122(self):
         new_sync_plan = self._make_sync_plan()
         result = SyncPlan.info({'id': new_sync_plan['id']})
         self.assertIsNotNone(result.get('enabled'))
+
+    # This Bugzilla bug is private. It is impossible to fetch info about it.
+    @stubbed('Unstab when BZ1279539 is fixed')
+    @tier1
+    def test_verify_bugzilla_1279539(self):
```

Comment:
The purpose of the test is to verify the case, specific to that BZ, but on the other hand, it may be considered as a negative test which does not rely on the specific BZ. Ok, i'll update the name, thanks for suggestion.